docs/Reference/CLI/CLI-Syntax.md

```diff
@@ -1809,3 +1813,4 @@ or clear your weak subjectivity settings.
 [weak subjectivity period]: ../../Concepts/Weak-Subjectivity.md
 [BeaconScan chain explorer]: https://beaconscan.com/ws_checkpoint
 [load new validators without restarting Teku]: ../../HowTo/Load-Validators-No-Restart.md
+[MainNet resources directory]: https://github.com/ConsenSys/teku/tree/master/util/src/main/resources/tech/pegasys/teku/util/config/mainnet
```

Comment:
Not sure it's a good idea to link to these resources here - they're slated to move in the near future and we could easily forget to update docs

Reply:
We have a nightly link check, so no risks we forget to update if they are broken.
But then if you think it will move soon we can either point to a better resource, if there's a stabler one, or include the examples directly in the doc, but then we will need you to tell us as soon as we need to update them. No solutions is perfect.
I don't think we should point to the root of the Teku repos, even if it will hopefully not change, as it will be too hard for users to find the files from there.

Reply:
The canonical place to look for config definitions is the eth2.0 spec repo [here](https://github.com/ethereum/eth2.0-specs/tree/master/configs).  Various testnet configs can be found [here](https://github.com/eth2-clients/eth2-testnets/tree/master/shared).

I'm not sure how helpful it is to include links - I think just explaining that you can use a single file, or split them up in a directory with subfiles named `phase0` or `altair` is really all the detail I was trying to convey.

Reply:
I like the idea to have a link to example when we talk about config files. It helps having a quick understanding of the content and capabilities. Your links are nice imo. @ConsenSys/protocol-pliny what do you think?

Reply:
Pliny recommends keeping the link.

Reply:
Use one of @mbaxter 's link then, they are closer to the source of truth and should not move too much.